hack/patch/README.md

```diff
@@ -7,14 +7,14 @@ Handles cherry-picks of PR(s) from etcd master to a stable etcd release branch a
 Set the `UPSTREAM_REMOTE` and `FORK_REMOTE` environment variables.
 `UPSTREAM_REMOTE` should be set to git remote name of `github.com/etcd-io/etcd`,
 and `FORK_REMOTE` should be set to the git remote name of the forked etcd
-repo (`github.com/${github-username}/etcd`). Use `git remotes -v` to
+repo (`github.com/${github-username}/etcd`). Use `git remote -v` to
 look up the git remote names. If etcd has not been forked, create
 one on github.com and register it locally with `git remote add ...`.
 
 
 ```
-export UPSTREAM_REMOTE=origin
-export FORK_REMOTE=${github-username}
+export UPSTREAM_REMOTE=upstream
```

Comment:
If user run `git clone https://github.com/etcd-io/etcd.git` which is instructed in our doc https://github.com/etcd-io/etcd/blob/master/Documentation/dl_build.md#build-the-latest-version, then 'origin is the upstream'?

Reply:
@jingyih thanks, and that's a good question. So yes, when you run the git clone that way, it will set `origin` to `https://github.com/etcd-io/etcd.git`. However, the cherrypick script is for the etcd developers/contributors, where one would fork it first and clone it so in that case origin is the forked repo. The `https://github.com/etcd-io/etcd.git` should be added using `git remove add` as we commonly do which is the upstream. That's how the script also documents it, https://github.com/spzala/etcd/blob/master/hack/patch/cherrypick.sh#L21 Also that's how 
README says https://github.com/spzala/etcd/tree/master/hack/patch#setup  I understand that you are just making sure that it's not confusing to the users of the script, hope the couple of links I mentioned above makes it clear for users? If you think it's still not clear, I am fine with modifying the doc a little more per what I have explained here. Thanks!

Reply:
I see. Thanks!

Reply:
@jingyih yrw, thank you!